src/main/java/fi/aalto/cs/apluscourses/model/Course.java

```diff
@@ -445,6 +470,45 @@ public void unregister() {
     // Subclasses may do things.
   }
 
+  @NotNull
+  public Map<String, String[]> getCourseReplInitialCommands() {
+    return replInitialCommands;
+  }
```

Comment:
Nit: I would rename this to `getReplInitialCommands`. Currently all of the private JSON helper methods include `Course` in the name and the public methods don't

Reply:
sharp eye! :eye: